tests/python/test_errors.py

```diff
@@ -326,8 +351,8 @@ def test_bad_type_run_script_multigpu(use_cluster, context):
     c = Client(None, use_cluster, logger_name=context)
     key = "my_script"
     fn_name = "phred"
-    inputs = ["list", "of", "strings"]
```

Comment:
You've updated test_bad_type_run_script_multigpu, but test_bad_type_run_script needs some love as well